tests/test_ontology_apis.py

```diff
@@ -0,0 +1,17 @@
+from opentargets_pharmgkb.ontology_apis import get_chebi_iri, get_efo_iri
+
+
+def test_get_chebi_iri():
+    # Exactly one exact match
+    assert get_chebi_iri('morphine') == 'http://purl.obolibrary.org/obo/CHEBI_17303'
+
+    # Multiple results but none that exactly match
+    assert get_chebi_iri('fluorouracil') is None
+
+
+def test_get_efo_iri():
+    # Exactly one high-confidence match
+    assert get_efo_iri('Lymphoma') == 'http://www.ebi.ac.uk/efo/EFO_0000574'
+
+    # No high-confidence matches
+    assert get_efo_iri('neoplasms') is None
```

Comment:
Out of curiosity, is the ``s`` what is bringing the confidence of the mapping down? After all, ``neoplasm`` ([``EFO:0000616``](http://www.ebi.ac.uk/efo/EFO_0000616)) does exist 🤔

Reply:
Yes that's right, just tried it and `get_efo_iri('neoplasm`)` does bring back `http://www.ebi.ac.uk/efo/EFO_0000616`.

We can be a bit more clever, e.g. do some stemming or something on the term, but for now I'm just using the exact same criteria by which we decide a term in ClinVar does not need manual curation, since IIRC we wanted to stay away from manual curation in PGKB initially.  I think we can re-evaluate this approach once we run this on the full dataset & get some counts.

Reply:
Good for now, and I assume it's minimal that these text variations are not picked, but I had to do this:
![Untitled](https://github.com/EBIvariation/opentargets-pharmgkb/assets/77288380/37ab9cb1-cb89-48f4-b065-20d43596e919)